kedro/framework/cli/starters.py

```diff
@@ -773,7 +812,7 @@ def _make_cookiecutter_args_and_fetch_template(
 
     tools = config["tools"]
     example_pipeline = config["example_pipeline"]
-    starter_path = "git+https://github.com/kedro-org/kedro-starters.git"
+    starter_path = _STARTERS_REPO
```

Comment:
Cannot comment on the proper line as it was not changed but the suggestion is what if we do not introduce the `starter_path` variable as it complicates the logic? So instead we can do:
```
    else:
        # Use the default template path for non PySpark, Viz or example options:
        cookiecutter_args, template_path
    return cookiecutter_args, _STARTERS_REPO
```

Reply:
That's a good point. To be honest, I'm not sure why this variable was there in the first place.

Reply:
Yes, I've seen it changed but I still prefer to return the original one instead. See the edited suggestion (I've missed a "return" and looked confusing)